h/static/styles/sidebar-tutorial.scss

```diff
@@ -0,0 +1,50 @@
+.sidebar-tutorial__header {
+  color: $color-cardinal;
+  font-weight: $title-font-weight;
+}
+
+/* We want an ordered list in which the numbers are aligned with the text
+   above (not indented or dedented), and wrapped lines in list items are
+   aligned with the first character of the list item (not the number, i.e
+   they're indented):
+
+       This is a list:
+
+       1. This is a list item.
+       2. This is a long list item
+          that wraps.
+       3. This is another list item.
+
+   What's more, we want the numbers to be a different color to the text of the
+   list items, which means we need to use ::before content for them.
+
+   This appears to be much harder than you'd think.
+
+   The code below comes from this Stack Overflow answer:
+   http://stackoverflow.com/questions/10428720/how-to-keep-indent-for-second-line-in-ordered-lists-via-css/17515230#17515230
+*/
+.sidebar-tutorial__list {
```

Comment:
The obvious approach that you'd hope you could take would be to specifically style the list marker. There _is_ a '::marker' pseudo-element for this but it isn't supported by any browsers AFAIK.